File: 9SummingOfDigit.cpp
#include<iostream>
using namespace std;

int summing(int n){
    int total  =0;
    while (n>0)
    {
        total += n%10;
        n/=10;
    }
    return total;
}

int main(){
    int n;
    cout<<"Enter the summing Number";
    cin>>n;
    cout<<summing(n);
}

File: 9SummingOfDigit.js
function summing(n){

    let string = String(n)
    let length = string.length;

    let totla = 0
    for(let i = 0 ;i < length; i++){
        totla += Number(string[i])
    }
    return totla
}

console.log('s', summing(1234))